docs/dev-guide/src/getting-started/getting-started.md

```diff
@@ -0,0 +1 @@
+# Getting started
```

Comment:
Looks like this stub can be deleted (or replaced). There is a `getting-started.md` in the parent folder that is linked to from the `SUMMARY.md`.